tests/reexport_external.rs

```diff
@@ -0,0 +1,30 @@
+use ion_schema::external::ion_rs::value::reader::{element_reader, ElementReader};
+use ion_schema::isl::isl_constraint::IslConstraint;
+use ion_schema::isl::isl_type::{IslType, IslTypeImpl};
+use ion_schema::isl::isl_type_reference::IslTypeRef;
+
+/// This test shows how the ion_schema integration with ion_rs can be used
+/// through a reexport. This means that consumers of ion_schema can use this
+/// integration without having to specify the exact dependency version.
+///
+/// See also: https://github.com/amzn/ion-schema-rust/issues/77.
```

Comment:
Since this issue is fixed by this PR, I'd say just include whatever explanation is in that issue inline instead of providing the link.